.github/workflows/e2e-tests.yml

```diff
@@ -12,23 +13,24 @@ on:
 
 jobs:
   e2e:
-    name: " " # GitHub UI is noisy when calling reusable workflows, so use whitespace for name to reduce noise
+    name: "E2E Tests"
```

Comment:
in [pr-environment-checks.yml](https://github.com/navapbc/template-infra/blob/rylew/e2e-containerize/.github/workflows/pr-environment-checks.yml#L68) we should set the name to " " from "Run E2E tests" so it will look cleaner in the PR i.e. it could be "CI App PR Environment Checks / / / E2E Tests (pull_request)" instead of "CI App PR Environment Checks / / Run E2E Tests / E2E Tests (pull_request)"

<img width="665" alt="image" src="https://github.com/user-attachments/assets/10c56e96-4860-4eb0-9757-ca7c7fd5e843">